.github/workflows/update_pr.yml

```diff
@@ -1,26 +0,0 @@
-name: update-pr
```

Comment:
Not sure why this is showing up as deleted, I undid my rename of it to `update-pull-request.yml` and pushed the change